Sources/Mistica/Components/Lists/ListCellContentView.swift

```diff
@@ -46,6 +46,7 @@ open class ListCellContentView: UIView {
     @frozen
     public enum CellAssetType: Equatable {
         case none
+        case url(URL, size: CGSize? = nil)
         case image(UIImage, size: CGSize? = nil)
```

Comment:
I would avoid having a new type here since this feature should be included in (be a subset of) the `.image` case.
```suggestion
        case url(URL, size: CGSize? = nil)
        case image(ImageCellAsset, size: CGSize? = nil)
```
and
```swift
public enum ImageCellAsset: Equatable {
     case .uiImage(UIImage)
     case .url(URL)
}
```
wdyt?

Reply:
Done :D